### PR TITLE
Added FullFlex date

### PR DESCRIPTION
**Pull request by FirestarJes**

### CODE REVIEW

File: source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/LocalDB/202209011008 Add FullFlexDate to GridAreaNew.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE [dbo].[GridAreaNew]
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/LocalDB/202209011008 Add FullFlexDate to GridAreaNew.sql:1]: Script missing in u, t and b env

File: source/Energinet.DataHub.MarketParticipant.Client/source/Energinet.DataHub.MarketParticipant.Client/Energinet.DataHub.MarketParticipant.Client.csproj [hunk 2]
```diff
@@ -27,7 +27,7 @@ limitations under the License.
 
   <PropertyGroup>
     <PackageId>Energinet.DataHub.MarketParticipant.Client</PackageId>
-    <PackageVersion>2.2.5$(VersionSuffix)</PackageVersion>
+    <PackageVersion>2.2.6$(VersionSuffix)</PackageVersion>
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.MarketParticipant.Client/source/Energinet.DataHub.MarketParticipant.Client/Energinet.DataHub.MarketParticipant.Client.csproj:30]: bump minor

Comment (michaelmajgaard) [reply source/Energinet.DataHub.MarketParticipant.Client/source/Energinet.DataHub.MarketParticipant.Client/Energinet.DataHub.MarketParticipant.Client.csproj]: rmember releas-notes.md

File: source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/LocalDB/202209011008 Add FullFlexDate to GridAreaNew.sql (hunk 1 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/LocalDB/202209011008 Add FullFlexDate to GridAreaNew.sql:1]: fixed

File: source/Energinet.DataHub.MarketParticipant.Client/source/Energinet.DataHub.MarketParticipant.Client/Energinet.DataHub.MarketParticipant.Client.csproj (hunk 2 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.Client/source/Energinet.DataHub.MarketParticipant.Client/Energinet.DataHub.MarketParticipant.Client.csproj]: Fixed

**Review by michaelmajgaard — APPROVED** [review]

**Pull request merged**